.gitattributes

```
@@ -1 +1,2 @@
-features/*.yml.dist linguist-language=YAML
+features/*.yml linguist-detectable
+features/*.yml.dist linguist-language=YAML linguist-generated
```

I think `linguist-generated` hides these files from diffs (like `package-lock.json`), but we look at them all the time.

```suggestion
features/*.yml.dist linguist-language=YAML
```

---

Done 🍏